modules/acquia_cms_image/acquia_cms_image.install

```diff
@@ -41,25 +40,138 @@ function acquia_cms_image_module_implements_alter(array &$implementations, strin
+        'id' => 'focal_point_scale_and_crop',
+        'weight' => 1,
+        'data' => [
+          'width' => 768,
+          'height' => 512,
+          'crop_type' => 'focal_point',
+        ],
+      ],
+    ],
+    'image.style.coh_x_large_landscape' => [
+      'ff9be98a-6552-4ae0-8687-8bf8e4dc61ef' => [
+        'uuid' => 'ff9be98a-6552-4ae0-8687-8bf8e4dc61ef',
+        'id' => 'focal_point_scale_and_crop',
+        'weight' => 1,
+        'data' => [
+          'width' => 1360,
+          'height' => 908,
+          'crop_type' => 'focal_point',
+        ],
+      ],
+    ],
+    'image.style.coh_xx_small_landscape' => [
+      '9846c8a0-70c1-4f9b-ae0a-53411f2a6c52' => [
+        'uuid' => '9846c8a0-70c1-4f9b-ae0a-53411f2a6c52',
+        'id' => 'focal_point_scale_and_crop',
+        'weight' => 1,
+        'data' => [
+          'width' => 160,
+          'height' => 120,
+          'crop_type' => 'focal_point',
+        ],
+      ],
+    ],
+    'image.style.coh_xx_large_landscape' => [
+      '4fc6f472-7800-4b98-a94d-0e29a3996610' => [
+        'uuid' => '4fc6f472-7800-4b98-a94d-0e29a3996610',
+        'id' => 'focal_point_scale_and_crop',
+        'weight' => 1,
+        'data' => [
+          'width' => 1600,
+          'height' => 1067,
+          'crop_type' => 'focal_point',
+        ],
+      ],
+    ],
+    'image.style.coh_large_super_landscape' => [
+      '8bbe9d4d-cd43-44e9-a209-457a4caaeee4' => [
+        'uuid' => '8bbe9d4d-cd43-44e9-a209-457a4caaeee4',
+        'id' => 'focal_point_scale_and_crop',
+        'weight' => 1,
+        'data' => [
+          'width' => 1024,
+          'height' => 480,
+          'crop_type' => 'focal_point',
+        ],
+      ],
+    ],
+    'image.style.coh_medium_super_landscape' => [
+      'a7be10be-726c-4b2a-8025-9a9b7dd85a00' => [
+        'uuid' => 'a7be10be-726c-4b2a-8025-9a9b7dd85a00',
+        'id' => 'focal_point_scale_and_crop',
+        'weight' => 1,
+        'data' => [
+          'width' => 768,
+          'height' => 360,
+          'crop_type' => 'focal_point',
+        ],
+      ],
+    ],
+    'image.style.coh_x_large_super_landscape' => [
+      'b2c98208-726a-44f9-99de-2624f33ac6bd' => [
+        'uuid' => 'b2c98208-726a-44f9-99de-2624f33ac6bd',
+        'id' => 'focal_point_scale_and_crop',
+        'weight' => 1,
+        'data' => [
+          'width' => 1360,
+          'height' => 640,
+          'crop_type' => 'focal_point',
+        ],
+      ],
+    ],
   ];
-  foreach ($image_styles as $image_style) {
-    $config_storage->write($image_style, $source_optional_dir->read($image_style));
+  $configFactory = \Drupal::service('config.factory');
+  foreach ($image_styles as $image_style => $effects) {
+    $config = $configFactory->getEditable($image_style);
+    $config->set('effects', $effects)->save();
+
+    // Flush updated image styles.
+    $imageStyle = \Drupal::entityTypeManager()->getStorage('image_style');
```

Comment:
@chandan-singh7929 We can add this code outside foreach.

Reply:
@chandan-singh7929 Also, we can reorganize this whole as:
```suggestion
    $image_styles = [
    "coh_small_square" => [568, 568],
    "coh_large_landscape" => [1024, 683],
    "coh_small_landscape" => [568, 352],
    "coh_medium_landscape" => [768, 512],
    "coh_x_large_landscape" => [1360, 908],
    "coh_xx_small_landscape" => [160, 120],
    "coh_xx_large_landscape" => [1600, 1067],
    "coh_large_super_landscape" => [1024, 480],
    "coh_medium_super_landscape" => [768, 360],
    "coh_x_large_super_landscape" => [1360, 640]
  ];
  foreach($image_styles as $image_style => $size) {
    $imageStyle = ImageStyle::load($image_style);
    if ($imageStyle instanceof ImageStyle) {
      list ($width, $height) = $size;
      $imageStyle->addImageEffect([
        'id' => 'focal_point_scale_and_crop',
        'weight' => 1,
        'data' => [
          'width' => $width,
          'height' => $height,
          'crop_type' => 'focal_point'
        ]
      ]);
      foreach ($imageStyle->getEffects() as $effect) {
        if ($effect->getPluginId() == "focal_point_crop") {
          $imageStyle->deleteImageEffect($effect);
          break;
        }
      }
      $imageStyle->save();
    }
  }
```

Reply:
Done